src/styles/partials/components/_accordion.scss

```diff
@@ -0,0 +1,97 @@
+.dg_accordion {
+  text-align: right;
+}
+
+.dg_accordion_buttonall-container {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.dg_button-link.dg_allitems {
+  padding-bottom: $padding-small;
+  padding-right: $base-spacing-unit;
+  //justify-content: flex-end;
+}
+
+.dg_accordion__collapsible {
+  color: $blue;
+  font-family: $default-heading-font;
+  font-size: $h2-font-size;
+  font-weight: $font-bold;
+  width: 100%;
+  border-bottom: $border-width-small $black solid;
+
+  &:first-of-type {
+    border-top: $border-width-small $black solid;
+  }
+
+  &:hover {
+    color: $vibrant-blue;
+  }
+
+  .dg_accordion__subheader {
+    font-size: $smallest-heading-font-size;
+    font-weight: $font-medium;
+    color: $gray;
+    padding-top: $padding-small;
+    text-transform: none;
+  }
+
+  .dg_accordion-item-body {
+    text-align: left;
+    font-size: $smallest-heading-font-size;
+    font-weight: $font-medium;
+    padding: $default-padding;
+    letter-spacing: 0;
+    color: $gray;
+    text-transform: none;
+  }
+
+  .dg_accordion-btn {
+    position: relative;
+    padding-top: $padding-large;
+    padding-bottom: $padding-small;
+    width: 100%;
+    text-align: left;
+    text-transform: uppercase;
+    height: auto;
+    font-family: $default-heading-font;
+    font-size: $small-heading-font-size;
+    font-weight: $font-bold;
+    cursor: pointer;
+
+    &:focus {
+      outline: none;
+    }
+
+    &:focus-within {
+      .dg_accordion_buttontext-holder {
+        @include default-focus-state;
+      }
+    }
+
+    &:after {
+      position: absolute;
+      padding-top: $padding-small;
+      padding-bottom: $padding-small;
+      font-size: $h1-font-size;
+      @include icon-content($icon-arrow-down);
+      top: 0;
+      right: 0;
+    }
+  }
+}
+
+.collapse {
```

Comment:
these probably need to move out of here eventually but not blocking, as i don't think we have a good place for them.